CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.12.0] - 2021-08-30
+
+### Changed
+
+* When we exit feature branch context manager, cleaned up the working directory.
+
```

Comment:
ideally, the changes to release the `0.12.0` version of this package will be a separate change